gub/specs/lilypond-test.py

```diff
@@ -38,26 +38,35 @@ def install (self):
         self.system('''
 echo %(targetdir)s > %(builddir)s/input/regression/out-test/AAA-prefix
 ''')
+        # Exclusions: from share/lilypond/current we only need fonts,
+        # and we don't need their sources.
         self.system ('''
-LD_PRELOAD= tar -C %(builddir)s -cjf %(test_ball)s input/regression/out-test
+LD_PRELOAD= tar -C %(builddir)s -chjf %(test_ball)s --exclude 'current/[^f]*' --exclude 'fonts/source' input/regression/out-test out/share/lilypond/current
 ''')
     def compile (self):
+        # TODO: It is concerning that these differences are introduced after
+        # building lilypond.  Shouldn't these things be established by
+        # configuring the lilypond build with certain options?
```

Comment:
I don't think it makes much since to fix it earlier, since TeX is not part of gub.  Adjusting the shared linker path is only necessary for building the documentation.

But maybe I don't fully understand your concerns, so please elaborate.

Reply:
My concern is that this section of GUB uses deeper knowledge of the internals of the lilypond build than it probably ought to, and also that there might be other side effects like the one this commit is intended to fix that remain to be discovered.  I don't want to have a long discussion about this, though.  I've removed the comment.